src/lib/components/Filteration.svelte

```
@@ -88,22 +88,22 @@
+					url.searchParams.delete(itemName + '_search');
+				} catch (e) {
+					console.error('Failed to update URL on popover close', e);
+				}
 				url.searchParams.delete(itemName + '_ids');
 				items.map((item: Filter) => {
 					url.searchParams.append(itemName + '_ids', item.id);
 				});
 
 				// reset pagination to first page when filters change
 				url.searchParams.set('page', '1');
+				goto(url, { keepFocus: true, invalidateAll: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)